.github/workflows/build-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and test
+on:
+  pull_request: { }
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: Set up Java environment
+        uses: actions/setup-java@v2.4.0
+        with:
+          distribution: temurin
+          java-version: 11
+          gpg-private-key: ${{ secrets.MAVEN_CENTRAL_GPG_SIGNING_KEY_SEC }}
```

Comment:
:question: Why do we need the Maven central stuff if we don't publish anything in this build?

Reply:
That's a good question. I just copied it from what we already had, but I'll remove it.